source/photo.html

```diff
@@ -51,9 +51,11 @@ <h1 class="main-header__heading main-header__heading--photo">Фото и вид
         <section class="gallery">
           <div class="photo-card photo-card--vastness">
             <a class="photo-card__link">
-              <img class="photo-card__image photo-card__image--vastness" src="img/photo-sunset-mobile@1x.jpg" width="280" height="280" alt="Фото просторов Седоны">
-              <img class="photo-card__image photo-card__image--vastness-tablet" src="img/photo-sunset-tablet@1x.jpg" width="648" height="400" alt="Фото просторов Седоны">
-              <img class="photo-card__image photo-card__image--vastness-desktop" src="img/photo-sunset-desktop@1x.jpg" width="1200" height="400" alt="Фото просторов Седоны">
+              <picture class="photo-card__picture">
+                <source media="(min-width: 1200px)" srcset="img/photo-sunset-desktop@1x.jpg" srcset="img/photo-sunset-desktop@2x.jpg">
+                <source media="(min-width: 768px)" srcset="img/photo-sunset-tablet@1x.jpg" srset="img/photo-sunset-tablet@2x.jpg">
+                <img class="photo-card__image photo-card__image--vastness" src="img/photo-sunset-mobile@1x.jpg" srcset="img/photo-sunset-mobile@2x.jpg" width="1200" height="400" alt="Фото просторов Седоны">
+              </picture>
```

Comment:
```suggestion
              <picture class="photo-card__picture">
                <source media="(min-width: 1200px)" srcset="img/photo-sunset-desktop@1x.jpg 1x, img/photo-sunset-desktop@2x.jpg 2x">
                <source media="(min-width: 768px)" srcset="img/photo-sunset-tablet@1x.jpg 1x, img/photo-sunset-tablet@2x.jpg 2x">
                <img class="photo-card__image photo-card__image--vastness" src="img/photo-sunset-mobile@1x.jpg" srcset="img/photo-sunset-mobile@2x.jpg 2x" alt="Фото просторов Седоны">
              </picture>
```

Остальные картинки поправь по аналогии.